tephra-hbase-compat-0.98/src/main/java/co/cask/tephra/hbase98/coprocessor/TransactionProcessor.java

```diff
@@ -151,7 +151,7 @@ public void preGetOp(ObserverContext<RegionCoprocessorEnvironment> e, Get get, L
     if (tx != null) {
       projectFamilyDeletes(get);
       get.setMaxVersions();
-      get.setTimeRange(TxUtils.getOldestVisibleTimestamp(ttlByFamily, tx), TxUtils.getMaxVisibleTimestamp(tx));
+      get.setTimeRange(0, TxUtils.getMaxVisibleTimestamp(tx));
```

Comment:
Even with existing pre-tx data, doesn't it make sense to have lower bound? It would be (current time - TTL) in actual milli seconds. If all pre-existing data is older than that, then we don't need to scan it.